docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -17,15 +17,16 @@ description: An asset definition is a description of how to compute the contents
   videos to get a quick look at asset definitions.
 </Note>
 
-An **asset** is an object in persistent storage, such as a table, file, or persisted machine learning model. A **asset definition** is a description, in code, of an asset that should exist and how to produce and update that asset.
+An **asset** is an object in persistent storage, such as a table, file, or persisted machine learning model. An **asset definition** is a description, in code, of an asset that should exist. It also usually describes how to update that asset.
 
 Asset definitions enable a declarative approach to data management, in which code is the source of truth on what data assets should exist and how those assets are computed.
 
 An asset definition includes the following:
 
 - An <PyObject object="AssetKey" />, which is a handle for referring to the asset.
-- A set of upstream asset keys, which refer to assets that the contents of the asset definition are derived from.
-- A Python function, which is responsible for computing the contents of the asset from its upstream dependencies and storing the results.
+- A set of upstream asset keys, which refer to assets that the contents of the asset are derived from.
+
+The primary kind of asset definition in Dagster is a _materializable_ asset definition. A materializable asset definition includes a Python function, which is responsible for computing the contents of the asset from its upstream dependencies and storing the results.
```

Comment:
Is this still accurate?

I thought 
AssetSpec + Computation = AssetDefinition.

Reply:
The way I have been thinking about it is AssetSpec + Optional Computation = AssetDefinition.

In the same way that AssetKey + Optional Description + Optional Metadata + Optional Tags etc. = AssetSpec.

Thoughts on this framing?